courses/migrations/0007_course_course_id.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2015-12-14 20:18
+from __future__ import unicode_literals
+
+import datetime
+from django.db import migrations, models
+from django.utils.timezone import utc
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('courses', '0006_author_name_optional'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='course',
+            name='course_id',
+            field=models.TextField(default=datetime.datetime(2015, 12, 14, 20, 18, 3, 515705, tzinfo=utc), help_text='course locator from edx', unique=True),
```

Comment:
I've updated this to do a data migration which fills in a random uuid. This isn't sensical for existing data, but if you have existing data right now.. you probably don't expect much by was of sense.